components/selection/selection-action.js

```diff
@@ -47,55 +43,47 @@ class Action extends LocalizeCoreElement(SelectionSubscriberMixin(ButtonMixin(Rt
 		`;
 	}
 
-	get selectionInfo() {
-		return super.selectionInfo;
-	}
-
-	set selectionInfo(value) {
-		super.selectionInfo = value;
-		this.nonInteractive = (this.requiresSelection && this.selectionInfo.state === SelectionInfo.states.none);
+	constructor() {
+		super();
+		this._selectionInfo = new SelectionInfoController(this);
 	}
 
 	connectedCallback() {
 		super.connectedCallback();
 		this.addEventListener('d2l-button-ghost-click', this._handleActionClick);
+		this._selectionInfo.hostConnected(); // remove with Lit 2
```

Comment:
Need to manually do this, but with Lit 2 all the lifecycle callbacks will be handled for us.